Assignment/solution_student/index.html

```diff
@@ -4,30 +4,83 @@
     <meta charset="UTF-8">
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
-    
-    <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
-    <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
-
-    <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/ico" href="images/favicon.png">
   </head>
   <body>
-
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <header class="clearfix">
+        <h1>Relaxr</h1>
+      <nav>
+       <ul>
+        <li><a href="#" class="nav-links">About</a></li>
+        <li><a href="#" class="nav-links">FAQ</a></li>
+        <li><a href="#" class="nav-links">Team</a></li>
+        <li><a href="#" class="nav-links">Contact Us</a></li>
+        <li><a href="#"class="nav-links">Blog</a></li>
+       </ul> 
+      </nav>
+     </header> 
+
+    <div class="main clearfix">
+      <div class="main-content">
+
+      <h2> How I implemented Relaxr in 2 weeks and changed my life.</h2>
+
+       <img class="main-images" src="images/blog_photo1.jpg">
+
+       <p class="first-character">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+       <!-- Is there a lorem ipsum text generator that you use? If so, please share your preference.-->
+
+       <p class="read-more">Read More ></p>
```

Comment:
It was my interpretation that these were links to the full blog posts, so these should be `<a>` tags